examples/i18n/README.md

```diff
@@ -0,0 +1,17 @@
+## I18N example
+
+How to support multiple languages in your web app based on [Fluent][url_project_fluent]. 
+Includes a language selector, some sample text and FTL strings demonstrating 
```

Comment:
Please attach a link to `FTL string` that points to docs - I have no idea what's that until I learn `fluent`. (In both READMEs)

Reply:
Sure. Link to FTL syntax guide added.